### DISCUSSION TITLE
Error: ENOENT: no such file or directory, open '.next/BUILD_ID' when deploying on Vercel

### DISCUSSION THREAD

**Discussion by diegoleft** [vercel/next.js#17142] [Help]
I'm trying to get the BUILD_ID with
`const BUILD_ID = fs.readFileSync(".next/BUILD_ID").toString();`

On my local is working well, but when I deploy on Vercel I'm getting this error
`Error: ENOENT: no such file or directory, open '.next/BUILD_ID'`

Is there a difference between a local Next js folder structure and Vercel?

**Comment by timneutkens** [comment]
`.next` does not exist yet as it's written at the end of the `next build`
